.circleci/config.yml

```diff
@@ -35,34 +35,23 @@ install_spython: &install_spython
       python --version
       python setup.py install
 
-install_python_3: &install_python_3
-  name: install Python 3 dependencies
-  command: | 
-      echo 'export PATH="$HOME/conda/bin:$PATH"' >> "$BASH_ENV"
+install_dependencies: &install_dependencies
+  name: install CI dependencies
+  command: |
+      PYTHON_VERSION=${PYTHON_VERSION:-3}
+      CONDA_PATH="$HOME/conda/Python${PYTHON_VERSION}"
+      echo 'export PATH="'"$CONDA_PATH"'/bin:$PATH"' >> "$BASH_ENV"
       source "$BASH_ENV"
-      if [ ! -d "$HOME/conda" ]; then
-          wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-          /bin/bash Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/conda
+      if [ ! -d "$CONDA_PATH" ]; then
+          CONDA_SCRIPT=Miniconda${PYTHON_VERSION}-latest-Linux-x86_64.sh
+          wget https://repo.anaconda.com/miniconda/$CONDA_SCRIPT
+          /bin/bash $CONDA_SCRIPT -b -p $CONDA_PATH
       else
-           echo "Miniconda 3 is already installed, continuing to build."
+           echo "Miniconda is already installed, continuing to build."
```

Comment:
We should add the PYTHON_VERSION here so when it's not installed, the user is still told what version is (should be).

Reply:
I added the `python --version` below to echo the version in both cases

Reply:
Yes! I saw right after my comment :)